packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts

```diff
@@ -0,0 +1,89 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { UTXO } from '../../../src/category/token'
+
+describe('Token', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  let from: string
+
+  async function setup (): Promise<void> {
+    from = await container.getNewAddress()
+
+    await createToken(from, 'DBTC')
+    await container.generate(1)
+
+    await createToken(from, 'DETH')
+    await container.generate(1)
+  }
+
+  async function createToken (address: string, symbol: string): Promise<void> {
+    const defaultMetadata = {
+      symbol,
+      name: symbol,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: address
+    }
+    await client.token.createToken({ ...defaultMetadata })
+  }
+
+  it('should mintTokens', async () => {
+    let tokenBalances = await client.account.getTokenBalances()
+
+    expect(tokenBalances.length).toStrictEqual(0)
+
+    const data = await client.token.mintTokens('5@DBTC')
```

Comment:
expect error while minting nonexistence coin

Reply:
<img width="180" alt="Screenshot 2021-06-01 at 1 01 43 PM" src="https://user-images.githubusercontent.com/7101806/120269152-d8726700-c2d9-11eb-9b09-5e3bbd1d0839.png">

Unable to get 10. 
105.6647572 - 95.6648398 = 9.9999174.

There should be fee involved.